File: PRAK505-2510817320004-RahmaSyaritaJaya.c
#include <stdio.h>

void Biodata(int a, char *b, char *c){
    int tahun_sekarang = 2025;
    printf("Perkenalkan Nama Saya : %s\n", b);
    printf("Umur Saya : %d\n", tahun_sekarang - a);
    printf("Saya Adalah Angkatan : %d\n", tahun_sekarang);
    printf("Asal Saya dari : %s\n", c);
}
int main() {
    int tahun_lahir;
    char A[20], B[15];
    scanf(" %d",&tahun_lahir);
    scanf(" %[^\n]%*c",&A);
    scanf(" %[^\n]%*c",&B);
    Biodata(tahun_lahir, A, B);
    return 0;
}

File: PRAK505-2510817320004-RahmaSyaritaJaya.py
def biodata(a, b, c):
    tahun_sekarang = 2025
    print(f"Perkenalkan Nama Saya : {b}")
    print(f"Umur Saya : {tahun_sekarang - a}")
    print(f"Saya Adalah Angkatan : {tahun_sekarang}")
    print(f"Asal Saya dari : {c}")


tahun_lahir = int(input())
nama = input()
asal = input()
biodata(tahun_lahir, nama, asal)